python/dgl/distributed/partition.py

```diff
@@ -339,10 +336,7 @@ def load_partition_book(part_config, part_id, graph=None):
         return RangePartitionBook(part_id, num_parts, node_map, edge_map, ntypes, etypes), \
                 part_metadata['graph_name'], ntypes, etypes
     else:
-        node_map = np.load(node_map)
-        edge_map = np.load(edge_map)
-        return BasicPartitionBook(part_id, num_parts, node_map, edge_map, graph), \
-                part_metadata['graph_name'], ntypes, etypes
+        raise Exception("Only RangePartitionBook is supported currently.")
```

Comment:
how about moving raise exception before handle: 
```
if not is_range_part: raise
#handle RangePartitionBook
```

Reply:
sure, move for fix.